src/services/mongoose/phone-code.types.d.ts

```diff
@@ -0,0 +1,11 @@
+interface IPhoneCodesRepository {
```

Comment:
this is used in app layer, should be defined in domain layer not in service layer (service layer only should have the implementation)

Reply:
is this what should be done https://github.com/GaloyMoney/galoy/pull/730/commits/921a61184dff4705b376c4c280a6d103713c6675?

Reply:
Did the fixing commit get dropped? This file is back in the latest diff.